### DISCUSSION TITLE
Deployed Node.js environment streaming is not handling chunked data stream, returns as one response

### DISCUSSION THREAD

**Discussion by MaitreyaBuddha** [vercel/community#4701] [Help]
### Summary

Using normal nodejs streaming with code like
```
  res.setHeader('Content-Type', 'application/octet-stream')
  res.setHeader('Transfer-Encoding', 'chunked')
  for await (const chunk of response.body as any) {
    res.write(chunk)
  }
  return res.status(200).end()
```
Will stream in the local dev environment, but not when deployed to Vercel. In fact, the 'Transfer-Encoding' header is being removed. The header was automatically included locally without being manually added.

How can I stream with an api without using the edge environment? The documentation says this is possible but only gived edge examples of code AFAICT.

### Steps to Reproduce

Create an API to stream content using nodejs serverless.

**Comment by Rizzist** [comment]
+1
